index.bs

```diff
@@ -689,6 +692,9 @@ The <dfn method for=CookieStore>set(|options|)</dfn> method steps are:
         |options|["{{CookieInit/domain}}"],
         |options|["{{CookieInit/path}}"], and
         |options|["{{CookieInit/sameSite}}"].
+    1. If the |options|["{{CookieInit/partitioned}}"] attribute is present and the user agent supports [cookie partitioning](https://github.com/WICG/CHIPS), then
+      1. If the current browsing context is a service worker, set the cookie's partition key be |origin|'s [=site=].
```

Comment:
... and "current browsing context" isn't the right term. I think "current global object" ?

Reply:
Also.. this is run after the cookie is set, which seems unusual. ISTM the partition key should be captured before "set a cookie" and passed into it.

Reply:
I think the partition key can be up to RFC6265bis to define, since the CookieStore API is not actually concerned with partition key values and the cookie RFC will (some day) specify how to select which partitioned cookies to include in a request. So I do not think this spec needs to be concerned with that?

Reply:
Yes... but https://wicg.github.io/cookie-store/#cookie-concept basically has "proxy" definitions for RFC6265bis terms, since I don't think we can link directly to them. We can add it later though.